ADOPTERS.md

```diff
@@ -11,9 +11,12 @@
 - [Asteria Aerospace](https://asteria.co.in)
   - "As we are a very modern technology company, we always want to use the cutting edge, open source technology to excel. All our microservices are modern in nature and so is their infrastructure. We are cloud agnostic and at the same time, we are fascinated by service meshes to empower us with secure, reliable, intelligent infrastructure. When we are in an Istio world, the need for managing the mesh was always been a gap. With **Meshery**, we believe we can nurture our service mesh to perform better and more efficiently."
 - [BootLabs](https://www.bootlabs.in/)
-  - "We are a true Boutique Tech Consulting partner who understands the pain points of all-sized company in terms of embracing Cloud.  Being obsessed about anything “CLOUD” and automation first, we drive our partner to success. We always strive to provide our customers ,the cutting edge technology and the trend is ServiceMesh now. We enable our Client to make their Micoservices more secure with Mesh. When we talk about ServiceMeshes, Meshery comes handy. We have seen the imact meshery has done for the Mesh deployment and Management and reduced the time to market "
+  - "We are a true Boutique Tech Consulting partner who understands the pain points of all-sized company in terms of embracing Cloud. Being obsessed about anything “CLOUD” and automation first, we drive our partner to success. We always strive to provide our customers ,the cutting edge technology and the trend is ServiceMesh now. We enable our Client to make their Microservices more secure with Mesh. When we talk about ServiceMeshes, Meshery comes handy. We have seen the impact meshery has done for the Mesh deployment and Management and reduced the time to market "
```

Comment:
Please add whitespace between Service and Mesh. Please capitalize every reference to Meshery.

Reply:
Add punctuation on the end of the sentence.

Add "in" to "Meshery comes handy"

Reply:
Implemented all the requested changes.